src/app/core/rest/project.service.ts

```diff
@@ -31,31 +31,59 @@ import {PermissionService} from './permission.service';
 export class ProjectService extends PermissionService {
 
   public getProjects(orgCode: string): Observable<Project[]> {
+    if (!this.hasOrganizationApiPrefix(orgCode)) {
+      return;
```

Comment:
I don't think it's the best solution, but it's a working one.
Why do you think it's better to do the checking in components?

>"We are basically hiding some underlying problem here"
I will try to investigate more about it.

>"Wouldn't it be better if this method throw exception in such cases?"
Fully agree and will change the `return` to `throw`.

Reply:
Because it does not make sense to call this method if the workspace is not set and the caller (component) is responsible for making sure it is not called in that state. So in my opinion, it is a problem in a component, don't you think?